Cargo.toml

```diff
@@ -59,6 +59,7 @@ postgres-replication = { git = "https://github.com/Mooncake-labs/rust-postgres.g
 postgres-types = { git = "https://github.com/Mooncake-Labs/rust-postgres.git", rev = "4fda9c1bece3b93359ca1eb2779458fc1c080eae", features = [
   "with-serde_json-1",
 ] }
+prometheus-client = "0.24.0"
```

Comment:
I'm not sure why do we need prometheus?

As we discussed yesterday, prom should be one of the detailed implementation for exporter, which should be enabled by features